search-enterprise/latest/using/index.md

```diff
@@ -124,11 +93,24 @@ appl?
 
 When performing a search that includes a wildcard character, it is best to wrap your search term in double quotation marks. This ensures all metadata and content are searched.
 
-### Search for conjunctions
+## Search for conjunctions
+
+Single terms, and phrases can be combined using `AND` in upper, lower, or mixed case.
+
+The `AND` operator is interpreted as "object1 and object2 will be returned".
+
+```text
+big banana
+big AND yellow AND banana
+TEXT:big TEXT:banana
```

Comment:
Same as above comment

Reply:
@tpage-alfresco  cheers

Reply:
This has been added based on work done with Angel